src/oracles/CoinbaseOracle.sol

```diff
@@ -28,6 +28,8 @@ contract CoinbaseOracle is IAggregatorV3SourceCoinbase {
      * @param _reporter The address of the reporter allowed to push price data.
      */
     constructor(uint8 _decimals, address _reporter) {
+        require(_reporter != address(0), "Invalid reporter address");
```

Comment:
There is another open PR to fix M-02 that hardcodes the reporter address in the CoinbaseOracle: https://github.com/UMAprotocol/oval-contracts/blob/c2580bf25b81cd3ac6cb6e9291164210d35e686f/src/oracles/CoinbaseOracle.sol#L31.

If we agree on these changes to fix M-02, L-07 will also be fixed, so probably these changes won’t be required anymore.

Reply:
agreed. ideally this is all hard coded as it should never change.